components/21-atoms/table/table.config.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = {
```

Comment:
(No need to create a variant if there's only 1 version of the component. Then it'll just be the `default` variant)

On the other hand, it might be useful to have some examples of different tables. One with column headers, one with row headers and one with both. 

Wdyt?

Reply:
Mmm good idea yes.

Reply:
Done.